### PR TITLE
Add container & component code coverage

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > Improving code coverage of our Supplemental Claim form; added code coverage to the non-component & container folders in a separate PR (#27831)
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits Decision Reviews
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

[#71499](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71499)

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

|         |   |
| ------- | ----- |
| Before  | Screenshot 2024-02-06 at 10 44 03 AM |
| After  | Screenshot 2024-02-07 at 4 51 00 PM |

## What areas of the site does it impact?

Supplemental Claims

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/appeals/995/tests/components/EvidencePrivateRecordsRequest.unit.spec.jsx [hunk 1]
```diff
@@ -65,6 +65,21 @@ describe('<EvidencePrivateRecordsRequest>', () => {
     expect(goSpy.called).to.be.false;
   });
 
+  it('should submit page', () => {
+    const goSpy = sinon.spy();
+    const data = { [EVIDENCE_PRIVATE]: true };
+    const { container } = render(
+      <div>
+        <EvidencePrivateRecordsRequest data={data} goForward={goSpy} />
```
Comment (rocio-desantiago) [inline src/applications/appeals/995/tests/components/EvidencePrivateRecordsRequest.unit.spec.jsx:73]: Is pressing the next button the action that checks the page is "submitted"?

Comment (Mottie) [reply src/applications/appeals/995/tests/components/EvidencePrivateRecordsRequest.unit.spec.jsx:73]: Yes

File: src/applications/appeals/995/tests/components/ITFBanner.unit.spec.jsx [hunk 2]
```diff
@@ -41,4 +42,27 @@ describe('ITFBanner', () => {
...
+    const { container } = render(
+      <ITFBanner status="itf-found" router={router} />,
+    );
+    $('va-button-pair', container).__events.secondaryClick(); // back
+    expect(router.push.args[0][0]).to.contain('/introduction');
+  });
+  it('should dismiss ITF page', () => {
+    const { container } = render(
+      <ITFBanner status="itf-found">
+        <div id="done">hello</div>
```
Comment (rocio-desantiago) [inline src/applications/appeals/995/tests/components/ITFBanner.unit.spec.jsx:62]: what's the `id=done` for?

**Review by rocio-desantiago — APPROVED** [review]

File: src/applications/appeals/995/tests/components/ITFBanner.unit.spec.jsx (hunk 2 above)
Comment (Mottie) [reply src/applications/appeals/995/tests/components/ITFBanner.unit.spec.jsx:62]: When the `ITFBanner` is dismissed, the content inside `<div id="done">...</div>` is rendered. We're checking that in the test below.

**Pull request merged**